### DISCUSSION TITLE
Invalidate Cached Crop

### DISCUSSION THREAD

**Discussion by marshallpenn** [SixLabors/ImageSharp.Web#136] [Q&A] [answered]
Is there anyway to invalidate a cached version or crop of an image? If a source image is updated it would be very useful to be able to force new cached images or crops. If it is just a matter of adding a new querystring value, is there anyway to delete/flush the now deprecated cached crops, or do they just build up over time? The intended implementation would be using Azure Blob storage for Source and cached images.

Thanks,
Marshall

**Comment by JimBobSquarePants** [comment, marked as answer]
There's no methods available as introducing them and no plan to do so as it also introduces opportunity for cache locks. 

Azure Blob Storage is for all intensive purposes limitless, allowing storage of 100s of terabytes of data.

**Comment by VILLAN3LL3** [comment]
@JimBobSquarePants Is there a sample how to use cache locks to achieve a behavior like intended by @marshallpenn ?
@marshallpenn Did you find a solution for your requirement? Can you share it? Thanks!

**Reply by binraider** [reply]
Hi Villan3ll3,

No i learned to just not worry about it in the end.
